docs/section21/Rule21-3.md

```diff
@@ -0,0 +1,43 @@
+
+# Boiler - Rule 21-3  
+
+**Rule ID:** 21-3  
+**Rule Description:** Heating hot water plant capacity shall be based on coincident loads.  
+**Rule Assertion:** B-RMR FluidLoop.heating_design_and_control: is_sized_using_coincident_load = True  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.2.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B-RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Check:**
+
+1. Rule is applicable if B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12: `PLACEHOLDER`
+
```

Comment:
Another applicability check is that proposed design doesn't have purchased heating

Reply:
Added.

Reply:
I don't think we need the first check (that there is at least one air-side system of Type 1, 5, etc.) since this is a prerequisite for Rule 21-1,

Reply:
Also, I don't think we have "NOT APPLICABLE" outcome... should be "UNDERTERMINED"

Reply:
As per discussion, each component will have 4 outcome, Pass, Fail, Undetermined and Not Applicable.